### ISSUE TITLE
[Woo POS] TalkBack allows to switch from Cart to Totals layout without Checking Out

### ISSUE THREAD

**Issue by staskus** [woocommerce/woocommerce-android#12498]
**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

TalkBack allows to switch from Cart to Totals layout without Checking Out.

I tested on an emulator, it may be not reproducible on a device.

**To Reproduce**
1. Enable TalkBack
2. Open POS
3. Use arrows to go to right
4. Notice it's possible to go from Product List to Cart and then switch to Totals view

**Video**

https://github.com/user-attachments/assets/298aa7c6-5575-46fa-867f-ba1f874e02a2

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active,
  or [Storefront](https://woocommerce.com/products/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Pixel Tablet Emulator
- Android version: API 34
- WooCommerce Android version:

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by kidinov** [comment]
I tried different approaches e.g.:

```
@Composable
private fun WooPosHomeScreen(
    state: WooPosHomeState,
    scrollState: ScrollState,
    productsWidthDp: Dp,
    cartWidthDp: Dp,
    totalsWidthDp: Dp,
    onHomeUIEvent: (WooPosHomeUIEvent) -> Unit,
) {
    val focusManager = LocalFocusManager.current

    LaunchedEffect(state.screenPositionState) {
        focusManager.clearFocus(force = true)
    }

    Box(
        modifier = Modifier
            .fillMaxSize()
            .background(WooPosTheme.colors.homeBackground)
    ) {
        Row(
            modifier = Modifier
                .horizontalScroll(scrollState, enabled = false)
                .fillMaxWidth()
                .focusGroup()
        ) {
            WooPosHomeScreenProducts(
                modifier = Modifier
                    .width(productsWidthDp)
                    .focusProperties {
                        canFocus = state.screenPositionState is WooPosHomeState.ScreenPositionState.Cart
                    }
            )
            WooPosHomeScreenCart(
                modifier = Modifier
                    .background(MaterialTheme.colors.surface)
                    .width(cartWidthDp)
                    .focusProperties {
                        canFocus = state.screenPositionState == WooPosHomeState.ScreenPositionState.Cart.Visible
                    }
            )
            WooPosHomeScreenTotals(
                modifier = Modifier
                    .width(totalsWidthDp)
                    .focusProperties {
                        canFocus = state.screenPositionState is WooPosHomeState.ScreenPositionState.Checkout
                    }
            )
        }

        WooPosHomeScreenToolbar(
            modifier = Modifier
                .padding(24.dp.toAdaptivePadding())
                .align(Alignment.BottomStart),
        )

        HandleProductsInfoDialog(state.productsInfoDialog, onHomeUIEvent)
    }
}
```

or with `focusable(true/false)`

But still, some internal elements of the subscreen could take the focus, which cause the scroll to another subscreen

Considering the impact, which is quite low, since a user needs to use a keyboard that is very rare. And nothing negative occurs—only another subscreen is visible, and the user just needs to click whatever they require to recover—I would propose closing this as high effort and very low impact.

@malinajirka @AnirudhBhat @samiuelson wdyt?

**Comment by AnirudhBhat** [comment]
Is this still reproducible from the latest `trunk`? I tried on both physical device and an emulator and couldn't reproduce it. Here's the video from my Pixel C tablet API 34 emulator.

https://github.com/user-attachments/assets/24565c94-5457-40e0-94c2-a8717a26ded4

**Comment by kidinov** [comment]
@AnirudhBhat I think I could reproduce it using "keyboard" focus change or something like that, but in the usual way, not really

**Comment by AnirudhBhat** [comment]
>I think I could reproduce it using "keyboard" focus change or something like that, but in the usual way, not really

Oh, I see. In that case, I think we could just close this issue as you suggested as this is something that would happen on an extremely rare condition and the time spending to fix this at this moment is not worth it.

**Comment by kidinov** [comment]
Ok lets close it!
